pkg/utils/values_patch.go

```diff
@@ -24,19 +22,13 @@ type ValuesPatch struct {
 	Operations []*ValuesPatchOperation
 }
 
-func NewValuesPatch() *ValuesPatch {
```

Comment:
This method is used somewhere in the Deckhouse

Reply:
Reverted